packs/panther.yml

```diff
@@ -7,9 +7,13 @@ PackDefinition:
     - Panther.SAML.Modified
     - Panther.Sensitive.Role
     - Panther.User.Modified
+    - Confluence.0DayIPs
```

Comment:
Action needed: 
* `Confluence.0DayIPs` -> This is a source IP 0day detection. I advise that we do not include it in the pack, as source IP based detections age rapidly. 
* `IOC.Sunburst` -> these detections are aging similarly, though they are hash or domain based. I would advise that we do not include these either.

Reply:
Noted. Will remove!